openmp/runtime/src/kmp.h

```diff
@@ -4384,7 +4384,7 @@ static inline bool __kmp_tdg_is_recording(kmp_tdg_status_t status) {
 
 KMP_EXPORT kmp_int32 __kmpc_start_record_task(ident_t *loc, kmp_int32 gtid,
                                               kmp_int32 input_flags,
-                                              kmp_int32 tdg_id);
```

Comment:
You should avoid `bool` in the compiler API (__kmpc*) and use fixed-sized integer instead. Can't you just put the reset flag somewhere into the input flags?

Reply:
Thanks for your review, `reset` has been moved into `input_flags` as requested